backend/tasks/banner.py

```diff
@@ -0,0 +1,72 @@
+from flask import request
+from flask_restful import Resource
+from flask import make_response, jsonify
+from flask_restful import reqparse
+from psycopg2.extras import RealDictCursor
+from pymysql.cursors import DictCursor
+import traceback
+import json
+import os
+
+class BannerTask(Resource):
+    def get(self):
+        try:
+            cur = request.db.cursor(cursor_factory=RealDictCursor) if os.environ.get('IS_LOCAL') else request.db.cursor(DictCursor)
```

Comment:
You can use a context manager to get the cursor so you don't explicitly have to call commit every time.